.copyright.hook

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import io, re
+import sys, os
+import subprocess
+import platform
+
+COPYRIGHT = '''
+  Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
2018

Reply:
clang-format removed.

Reply:
Great - yes this is exactly it needs to be dynamically replaced